libs/template/templates/default-sql/library/versions.tmpl

```diff
@@ -0,0 +1,7 @@
+{{define "latest_lts_dbr_version" -}}
+  13.3.x-scala2.12
```

Comment:
shouldn't this be `14.3 LTS` by now? Same for DBConnect.

Reply:
Let's update all templates at the same time with that change. And I would honestly let 14.3 LTS bake a while longer. It still has issues like https://databricks.atlassian.net/browse/ES-1000398.